src/components/ProjectCard/ProjectCard.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import "./ProjectCard.css";
+
+import { Link } from "react-router-dom";
+
+export default function ProjectCard(props) {
+    const { year, title, img, technologies, bgColor } = props;
+    return (
+        <div
+            style={{ backgroundColor: bgColor }}
+            className="project-card-container"
+        >
+            <div className="project-card-info">
+                <p>{year}</p>
+                <strong>
+                    <h2>{title}</h2>
+                </strong>
+                <p>Full Stack Development</p>
```

Comment:
you should add this to the data file .. maybe soon you will add a project that is forntend only